requirements.txt

```diff
@@ -1,12 +1,12 @@
-boto3==1.17.5
```

Comment:
I see only two major versions, it seems they didn't break your code right?

Reply:
No, I just try to update every package to the latest version to see if it fix the issue